ci/ciimage/arch/install.sh

```diff
@@ -15,6 +15,7 @@ pkgs=(
   doxygen vulkan-validation-layers openssh mercurial gtk-sharp-2 qt5-tools
   libwmf cmake netcdf-fortran openmpi nasm gnustep-base gettext
   python-lxml hotdoc rust-bindgen qt6-base qt6-tools wayland wayland-protocols
+  debugedit glib2-devel
```

Comment:
debugedit should not be needed -- fixed better as f009ccb81bde95a414ca071aadfe24b44b39398b.

Reply:
rebased, thanks!